channel-operation.asciidoc

```diff
@@ -1,6 +1,13 @@
 [[channel_operation]]
 == Channel Operation and Payment Forwarding
 
+In this chapter we will bring together payment channels and HTLCs. As mentioned in <<payment_channels>>, a simple payment over a payment channel uses HTLCs.
+
+Even though it is possible to send payments across a payment channels simply by updating the channel balances and creating new commitment transactions, the Lightning protocol uses HTLCs. The reason for this is to maintain the same protocol design regardless of whether a payment is only one hop (across a single payment channel), or several hops (routed across multiple payment channels).
```

Comment:
There is a typo here. "Even though it is possible to send payments across a payment channel_**s**_ simply by updating the channel balances"